content/tagging/using_tags.md

```diff
@@ -12,105 +12,271 @@ further_reading:
+[2]: /graphing/event_stream/
+[4]: /graphing/dashboards/
+[5]: /graphing/dashboards/template_variables/
+
+{{% /tab %}}
+{{% tab "Examples" %}}
+
+Here is an example of tags using the time-series chart editor. For the first screenshot, no tags have been applied, and we're observing average CPU across all hosts:
+
+{{< img src="tagging/using_tags/Tags_1.png" alt="Tags_1" responsive="true" style="width:75%;">}}
+
+Next, we've updated the editor to include a tag (`region:eastus`) in the **from** textbox that enables Datadog to look at CPU across the US East Region. We've used region as an example, but you could use any arbitrary tag, including application, service, environment, etc.
+
+{{< img src="tagging/using_tags/Tags_2.png" alt="Tags_2" responsive="true" style="width:75%;">}}
+
+Finally, we've used the second empty field (the **avg by** textbox) to show an individual timeseries line for each `host`. Now we're seeing server CPU for individual hosts running in the US East Region.
+
+{{< img src="tagging/using_tags/Tags_3.png" alt="Tags_3" responsive="true" style="width:75%;">}}
+
+If needed, you can add additional tags to narrow down the scope even further - for example, hosts in `region:eastus` and `env:production`. Tags are extremely powerful, and they are used throughout Datadog. They can be applied to all core elements.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Infrastructure
+
+To filter the [Host Map][6], [Infrastructure List][7], [Containers][8], and [Processes][9] enter a tag in the **Filter by** textbox at the top of the page. You can also group hosts / containers by entering the key portion of the tag in the **Group by** textbox. So, if you enter `service` in the group box, you see each service as a group heading. 
+
+{{< tabs >}}
+{{% tab "Host Map" %}}
+
+Under this section, use tags to filter or group Hosts:
+
+{{< img src="tagging/using_tags/hostmaptags.png" alt="Host Map Tags" responsive="true" style="width:80%;">}}
+
+Or Containers:
+
+{{< img src="tagging/using_tags/containermaptags.png" alt="Container Map Tags" responsive="true" style="width:80%;">}}
+{{% /tab %}}
+
+{{% tab "Infrastructure List" %}} 
+
+{{< img src="tagging/using_tags/infrastructuretags.png" alt="Tags in the Infrastructure List" responsive="true" style="width:80%;">}}
+{{% /tab %}}
+
+{{% tab "Containers" %}}
+
+{{< img src="tagging/using_tags/livecontainertags.png" alt="Live Container Tags" responsive="true" style="width:80%;">}}
+{{% /tab %}}
+
+{{% tab "Processes" %}}
+
+{{< img src="tagging/using_tags/liveprocessestags.png" alt="Live Process Tags" responsive="true" style="width:80%;">}}
+{{% /tab %}}
+{{< /tabs >}}
+
+## Monitors
+
+{{< tabs >}}
+{{% tab "Manage Monitors" %}}
+
+To filter monitors by [assigned tags][1], use the search bar or facet checkboxes. The search bar format is `tag:key:value`, for example `tag:service:coffee-house`. **Note** monitor tags are different and separate from metric tags.
```

Comment:
since we are displaying a placeholder we should follow the placeholders guidelines: 
`tag:key:value` => `tag:<KEY>:<VALUE>`